streaming_providers/realdebrid/utils.py

```diff
@@ -46,21 +46,16 @@ def get_video_url_from_realdebrid(
             raise ProviderException(
                 "Torrent limit reached. Please try again later.", "torrent_limit.mp4"
             )
-        if info_hash in response["list"]:
```

Comment:
I think this is needed. Sometimes the torrent is in active.

Reply:
My understanding is as follows:

1. `get_available_torrent` returns a valid `torrent_info` if the torrent is available in any state in RD.
2. If `torrent_info` is none, it means the torrent is not available in RD and has to be added.
3. If the torrent limit is not yet reached, we add the torrent. Else, we raise an exception.

When does the case you mentioned above occur?

Reply:
Okay that should be fine. let's keep the `if info_hash in response["list"]:` condition and raise correct exception message.

Reply:
OK. I have reverted the lines.